uaa/src/main/resources/uaa.yml

```diff
@@ -2,10 +2,6 @@
 # if any of these exist:
 # [$UAA_CONFIG_URL, $UAA_CONFIG_PATH/uaa.yml, $CLOUDFOUNDRY_CONFIG_PATH/uaa.yml]
 
```

Comment:
This file should not be change with the merge. Revert this change. 

Also, this setting should be added to uaa-docker.yml to support JwtBearerGrantAT as in [uaa-docker.yml#L5-L8](https://github.com/GESoftware-CF/uaa/blob/predix_prunes/jwt_grant_75.6.0/uaa/src/main/resources/uaa-docker.yml#L5-L8) in /predix_prunes/jwt_grant_75.6.0 branch

Reply:
All tests pass without the device config lines (see link to passing job in comment). But yeah, there are references to this device configuration, but no tests fail. I guess there are missing tests - anyway, I will leave the lines alone.